packaging/commands/zypper.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2015 Canonical Ltd.
+// Copyright 2015 Cloudbase Solutions SRL
+// Licensed under the LGPLv3, see LICENCE file for details.
+// Created from yum.go
+
+package commands
+
+const (
+	// OpenSUSESourcesDir is the default directory in which openSUSE sourcefiles
+	// may be found.
+	OpenSUSESourcesDir = "/etc/zypp/repos.d"
+
+	// TODO: Something similar in zypper?
+	// CentOSYumKeyfileDir is the default directory for yum repository keys.
+	// CentOSYumKeyfileDir = "/etc/pki/rpm-gpg/"
+
+	// OpenSUSECredentialsDir is the sirectory for credentials.
+	OpenSUSECredentialsDir = "/etc/zypp/credentials.d"
+
+	// OpenSUSESourcesFile is the default file which lists all core sources
+	// for zypper packages on OpenSUSE.
+	OpenSUSESourcesFile = "/etc/zypp/repos.d/repo-oss.repo"
+
+	// ZypperConfigFile is the default configuration file for yum settings.
+	ZypperConfigFilePath = "/etc/zypp/zypp.conf"
+
+	//OpenSUSE proxy settings
+	OpenSUSEProxy = "/etc/sysconfig/proxy"
+)
+
+const (
+	// Zypper command for managing packages and repos
+	//		--quiet to only show errors
+	//		--non-interactive to install without asking packages
+	zypper = "zypper --quiet --non-interactive"
+
+	// TODO: is useful?
```

Comment:
I'd like to switch proxySettingsFormat to per-scheme strings (i.e. httpProxySettingsFormat, httpsProxySettingsFormat, ftpProxySettingsFormat). Until then, it's fine to have this as a constant; feel free to drop the TODO.